### ISSUE TITLE
Sentence in "About this page" for Common Failures techniques

### ISSUE THREAD

**Issue by carlosapaduarte** [w3c/wai-wcag-supporting-documents-redesign#62]
The sentence reads "Techniques are examples of ways to meet a WCAG success criterion."
But Common Failures techniques are examples of ways to fail a SC, not to meet. The sentence should be in accordance.

**Comment by shawna-slh** [comment]
HI Carlos, Looks like it was just because we were looking at incomplete or wrong prototype. In in https://w3c.github.io/wai-wcag-supporting-documents-redesign/2020-07-15-prototype-failure.html I found this:
> Failure Techniques show ways in which criteria are considered “not met”.
